index.html

```diff
@@ -4,37 +4,172 @@
       <h1>
-      I'm Uchechi<br>
-      Glad to see<br>
-      you!
-      </h1>
-      <p>
-      I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-      </p>
-      <h3>LETS CONNECT</h3>
-      <ul>
-        <li><a href="#"><img src="images/Tweeter.png"></a></li>
-        <li><a href="#"><img src="images/Linkedin icon.png"></a></li>
-        <li><a href="#"><img src="images/m.png"></a></li>
-        <li><a href="#"><img src="images/Github.png"></a></li>
-        <li><a href="#"><img src="images/tech.png"></a></li>
+        I'm Uchechi<br>
+        Glad to see<br>
+        you!
+        </h1>
+        <p>
+        I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h3>LETS CONNECT</h3>
+        <ul class="social">
+          <li><a href="#"><img src="images/Tweeter.png"></a></li>
+          <li><a href="#"><img src="images/Linkedin icon.png"></a></li>
+          <li><a href="#"><img src="images/m.png"></a></li>
+          <li><a href="#"><img src="images/Github.png"></a></li>
+          <li><a href="#"><img src="images/tech.png"></a></li>
+        </ul>
+    </main>
+
+  </header>
+  <main class="cards-1">
+    <section id="first-work">
+      <div class="snap">
+        <img src="images/Snapshoot Portfolio.png" width="311px" height="220px" class="snap">
+      </div>
+      <section class="all">
+        <h5 class="title">Tonic</h5>
+        <ul class="back-end">
+        <li id="canopy">CANOPY</li>
+        <li><img src="images/Counter.svg" class="dev">Back End Dev</li>
+        <li><img src="images/Counter.svg" class="dev">2015</li>
+        </ul>
+        <p class="des">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="code">
+          <li id="htm">html</li>
+          <li id="cs">css</li>
+          <li id="js">javascript</li>
+      </ul>
+        <a href="#"><button type="button" class="box">
+            <p>See Project</p>
+        </button></a>
+      </section>
+    </section>
+  </main>
+  <main class="cards-2">
+    <section id="second-work">
+      <div class="snap">
+        <img src="images/avail.png" width="311px" height="220px" class="snap">
+      </div>
+      <section class="all">
+        <h5 class="title">Multi-Post <br> Stories</h5>
+        <ul class="back-end">
+        <li id="canopy">CANOPY</li>
+        <li><img src="images/Counter.svg" class="dev">Back End Dev</li>
+        <li><img src="images/Counter.svg" class="dev">2015</li>
+        </ul>
+        <p class="des">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="code">
+          <li id="htm">html</li>
```

Comment:
please make sure your whole work section is inside a single `<section>` tag,  and use Display [grid](https://css-tricks.com/snippets/css/complete-guide-grid/) for the layout of this section, additionally we have to style each card inside the work section with [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
for thw Works section :
#### Works

1. The container of the section is a `section` tag
1. The main container of the projects has a `display:grid`

you can read more about [section tag here](https://www.w3schools.com/tags/tag_section.asp) and[ semantic html here](https://www.w3schools.com/html/html5_semantic_elements.asp)